yosys-plugin/churchroad.cc

```diff
@@ -1483,7 +1483,7 @@ struct LakeroadWorker
 					op_str = "(Or)";
 				else if (cell->type == ID($xor))
 					op_str = "(Xor)";
-				else if (cell->type == ID($shr))
```

Comment:
@gussmith23 Here, both `shr` and `shiftx` will map to `(Shr)`, mainly because the distinction between the two in the [Yosys documentation](https://yosyshq.readthedocs.io/projects/yosys/en/latest/yosys_internals/formats/cell_library.html#binary-operators) seems like something we can't represent in Churchroad anyways:

> The $shift cell performs a right logical shift if the second operand is positive (or unsigned), and a left logical shift if it is negative. The $shiftx cell performs the same operation as the $shift cell, but the vacated bit positions are filled with undef (x) bits

Reply:
Documented in #26 .